sample-xml/README.md

```diff
@@ -0,0 +1,51 @@
+# Android XML Sample
+
+A sample project generated from our template using XML layouts
+
+## Setup
+
+- Clone the project
+- Run the project with Android Studio
+
+## Linter and static code analysis
+
+- Lint:
+
+```
+$ ./gradlew lint
+```
+
+Report is located at: `./app/build/reports/lint/`
+
+- Detekt
+
+```
+$ ./gradlew detekt
+```
+
+Report is located at: `./build/reports/detekt`
+
+## Testing
+
+- Run unit testing:
+
+```
+$ ./gradlew app:testStagingDebugUnitTest
+$ ./gradlew data:testDebugUnitTest
+$ ./gradlew domain:test
+```
+
+- Run unit testing with coverage:
+
+```
+$ ./gradlew jacocoTestReport
+```
+
+Report is located at: `./app/build/reports/jacoco/`
+
+## Build and deploy
+
+For `release` builds, we need to provide release keystore and signing properties:
+
+- Put the `release.keystore` file at root `config` folder.
+- Put keystore signing properties in [signing.properties](https://github.com/nimblehq/android-templates/blob/develop/sample-xml/signing.properties).
```

Comment:
FYI: I ended up removing the link, for the sake of simplicity (https://github.com/nimblehq/android-templates/pull/277)

Reply:
@Tuubz Oh, I only followed your suggestion [here](https://github.com/nimblehq/android-templates/pull/256#discussion_r942036435) 😅

Reply:
Yeah, sorry for that. 

I ended up removing the link, I didn't want to overcomplicate the link 🙇